docs/components/motor/tmc5072.md

```diff
@@ -106,7 +106,7 @@ The following attributes are available for `TMC5072` motors:
 <!-- prettier-ignore -->
 | Name | Type | Inclusion | Description |
 | ---- | ---- | --------- | ----------- |
-| `spi_bus` | string | **Required** | The board [SPI bus](/components/board/#spis) over which the TMC chip communicates with the board. |
+| `spi_bus` | string | **Required** | The board SPI bus over which the TMC chip communicates with the board. |
```

Comment:
Mild preference to remove the word "board" and have this just be "The SPI bus over which..." but I don't feel strongly. We already mention the board at the end of the sentence.

Reply:
Good with me!